admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -150,17 +177,15 @@ export const ApplicationPage: React.FunctionComponent = () => {
                                                     type='text'
                                                     value={ eventType.displayName }
                                                     onChange={ handleChange }
-                                                    id='display-name'
-                                                    name="displayName" /></FormGroup>
```

Comment:
Isn't the name needed for the edit ?

Reply:
yes, didn't delete that on purpose. adding back.